packages/edit-site/src/components/sidebar-edit-mode/page-panels/edit-template.js

```diff
@@ -34,7 +35,7 @@ export default function EditTemplate() {
 				queryArgs
 			),
 			title: template?.title,
-			blocks: template?.blocks,
+			content: template?.content,
```

Comment:
I think we should retrieve both blocks and content and prefer blocks if defined otherwise, you'll get an outdated preview.